tools/autodeployment/deploy_stage.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Shutdown
+cd ../telescope
+
+docker-compose -f docker-compose-staging.yml down
+
+docker system prune -af
+
+
+# Delete and clone
+cd ..
+
+rm -rf telescope
+
+git clone https://github.com/Seneca-CDOT/telescope.git
```

Comment:
An improvement we could make in a future PR is to checkout the specific commit sha that was just pushed vs. whatever is currently on `master`.  It's unlikely they will be different, but if they are, we could have odd bugs.